github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-postsubmits.yaml

```diff
@@ -65,3 +65,145 @@ postsubmits:
+            start_libvirtd.sh && cd cluster-provision/images/vm-image-builder &&
+            export KUBEVIRTCI_TAG="$(date +"%y%m%d%H%M")-$(git rev-parse --short HEAD)“ &&
+            ./publish-multiarch-containerdisk.sh fedora-with-test-tooling kubevirtci quay.io &&
+            echo $KUBEVIRTCI_TAG > latest &&
+            gsutil cp ./latest gs://kubevirt-prow/release/kubevirt/kubevirtci/fedora-with-test-tooling-latest
+          env:
+          - name: WITHCONSOLE
+            value: "no"
+          # docker-in-docker needs privileged mode
+          securityContext:
+            privileged: true
+          volumeMounts:
+          - mountPath: /lib/modules
+            name: modules
+            readOnly: true
+          - mountPath: /dev
+            name: devices
+          resources:
+            requests:
+              memory: "8Gi"
+    - name: publish-fedora-realtime
+      branches:
+      - main
+      always_run: false
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      decoration_config:
+        timeout: 2h
+      run_if_changed: cluster-provision/images/vm-image-builder/.*\.sh$ | cluster-provision/images/vm-image-builder/fedora-realtime/.*
+      max_concurrency: 1
+      extra_refs:
+      - org: kubevirt
+        repo: project-infra
+        base_ref: main
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-gcs-credentials: "true"
+        preset-kubevirtci-quay-credential: "true"
+        preset-podman-in-container-enabled: "true"
+      cluster: prow-workloads
+      spec:
+        affinity:
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+              - matchExpressions:
+                - key: kubernetes.io/hostname
+                  operator: NotIn
+                  values:
+                  - bare-metal-9
+        nodeSelector:
+          type: bare-metal-external
+        volumes:
+        - hostPath:
+            path: /lib/modules
+            type: Directory
+          name: modules
+        - hostPath:
+            path: /dev
+            type: Directory
+          name: devices
+        containers:
+        - image: quay.io/kubevirtci/vm-image-builder
+          command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/bash"
+          - "-c"
+          - >
+            cat $QUAY_PASSWORD | docker login --username $(<$QUAY_USER) --password-stdin quay.io &&
+            start_libvirtd.sh && cd cluster-provision/images/vm-image-builder &&
+            export KUBEVIRTCI_TAG="$(date +"%y%m%d%H%M")-$(git rev-parse --short HEAD)“ &&
+            ./publish-multiarch-containerdisk.sh fedora-realtime kubevirtci quay.io &&
+            echo $KUBEVIRTCI_TAG > latest &&
+            gsutil cp ./latest gs://kubevirt-prow/release/kubevirt/kubevirtci/fedora-realtime-latest
```

Comment:
This will probably break automation that we have for bumping kubevirtci in kubevirt/kubevirt - there won't be corresponding kubevirtci providers to match the tag in  the `latest` file. 

The automation for this PR - https://github.com/kubevirt/kubevirt/pull/9616

https://github.com/kubevirt/kubevirt/blob/main/hack/bump-kubevirtci.sh

Reply:
It will not influence the current process, as it upload to file `fedora-realtime-latest` and `fedora-with-test-tooling` rather than `latest` via the following commend.
```
 gsutil cp ./latest gs://kubevirt-prow/release/kubevirt/kubevirtci/fedora-realtime-latest
```
And the script `bump-kubevirtci.sh` is checking the file `latest` in the google cloud.
https://github.com/kubevirt/kubevirt/blob/29526b6cd255c4e3c0bf13995628559655733fe4/hack/bump-kubevirtci.sh#LL8C54-L8C55

Reply:
Yes you are right - should be fine.

Reply:
In case misunderstanding, I will re-write the lines like the following.
```
            echo $KUBEVIRTCI_TAG > fedora-realtime-latest &&
            gsutil cp ./fedora-realtime-latest gs://kubevirt-prow/release/kubevirt/kubevirtci/fedora-realtime-latest
```